frontend/packages/console-app/src/components/nodes/NodeLogs.tsx

```diff
@@ -0,0 +1,105 @@
+import * as React from 'react';
+import { Checkbox } from '@patternfly/react-core';
+import classnames from 'classnames';
+import { useTranslation } from 'react-i18next';
+import {
+  Dropdown,
+  getQueryArgument,
+  LoadingBox,
+  setQueryArgument,
+} from '@console/internal/components/utils';
+import { modelFor, NodeKind, resourceURL } from '@console/internal/module/k8s';
+
+type NodeLogsProps = {
+  obj: NodeKind;
+};
+
+const NodeLogs: React.FC<NodeLogsProps> = ({ obj: node }) => {
+  const dropdownItems = {
+    journal: 'journal',
+    'openshift-apiserver': 'openshift-apiserver',
+    'kube-apiserver': 'kube-apiserver',
+    'oauth-apiserver': 'oauth-apiserver',
+  };
+  const queryArgument = 'api';
+  const [currentKey, setCurrentKey] = React.useState(
+    getQueryArgument(queryArgument) || dropdownItems.journal,
+  );
+  const [isLoading, setIsLoading] = React.useState(true);
+  const [content, setContent] = React.useState('');
+  const [isWrapLines, setIsWrapLines] = React.useState(false);
+  const { t } = useTranslation();
+  const onChange = (newAPI) => {
+    setCurrentKey(newAPI);
+    setQueryArgument(queryArgument, newAPI);
+    setIsLoading(true);
+  };
+  React.useEffect(() => {
+    const {
+      kind,
+      metadata: { name, namespace: ns },
+    } = node;
+    const baseURL = `proxy/logs/${currentKey}`;
+    const nodeLogURL = resourceURL(modelFor(kind), {
+      name,
+      ns,
+      path: currentKey === 'journal' ? baseURL : `${baseURL}/audit.log`,
```

Comment:
We shouldn't always just get the audit log. There might be other logs under `openshift-api` that we care about. I do think we need to make this a free form input.

Reply:
Have been thinking about this and am wondering if trying to add the logs to the UI is a mistake?  With the CLI, you first have to list the logs in order to get the log names for each of the three APIs.  Once you have the names (which are dynamic), you can then view the one you are interested in.  So even if we made it a free form input, you'd still have to know the log name (e.g., `audit-2021-03-09T00-12-19.834.log` or the generic `audit.log`).  Unless there is an API that will give us the log names?

Reply:
> Unless there is an API that will give us the log names?

Duh.  It is the same API that gives the logs.

Reply:
Yeah, I see what you're saying. Maybe we restrict this to just `journal` for the initial PR.

I could see adding a tree component to be able to drill down to the log you want eventually. We'd need some design input.